src/frontend/components/DashboardVideoLiveControlPane/inputs/DashboardVideoLiveWidgetTextInput/index.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { DashboardVideoLiveWidgetTextInput } from '.';
+
+const setValueMock = jest.fn();
```

Comment:
this mock could be a real function saving the value enter by the user and then you can check that the value stored in it is the one expected.
Something like this

```suggestion
let inputTextValue;
const setValueMock = (value) => inputTextValue = value;
```

In the beforeEach you reset `inputTextValue` variable.

Reply:
I changed it